definitions/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -1235,6 +1235,12 @@
     "UserRole": "caseworker-employment-legalrep-solicitor",
     "CRUD": "CRU"
   },
+  {
+    "CaseTypeId": "ET_EnglandWales",
+    "CaseEventID": "claimantTSE",
+    "UserRole": "caseworker-employment-legalrep-solicitor",
+    "CRUD": "CRU"
+  },
```

Comment:
I think this should be D?
A legal rep should not be able to access the claimantTSE event unless they have the claimantsolitictor role?

Reply:
resolved in https://github.com/hmcts/et-ccd-definitions-englandwales/pull/946/commits/4f772d528784427c0849821e05d607efda577c1a